src/main/java/edu/agh/wfiis/designpatterns/patternsUsageExample/DesignPatternsManager.java

```diff
@@ -59,21 +43,24 @@ public String generateDesignPatternsCourseworkForStudents(){
         return "very difficult coursework";
```

Comment:
Method generateDesignPattersCourseworkForStudents should be extracted to another class because there is no bound between this functionality and managing design patterns. Cohesion is low.